regtests/client/python/cli/polaris_cli.py

```diff
@@ -108,8 +112,13 @@ def _get_client_builder(options):
                             f' Alternatively, you may set the environment variables {CLIENT_ID_ENV} &'
                             f' {CLIENT_SECRET_ENV}.')
         # Authenticate accordingly
-        polaris_management_url = f'http://{options.host}:{options.port}/api/management/v1'
-        polaris_catalog_url = f'http://{options.host}:{options.port}/api/catalog/v1'
+        if options.base_url:
+            polaris_management_url = f'{options.base_url}/api/management/v1'
+            polaris_catalog_url = f'{options.base_url}/api/catalog/v1'
+        else:
```

Comment:
Similar to access-token vs client-id/client-secret, we should validate that both aren't provided.

Reply:
Done